docs/API.md

```diff
@@ -175,3 +176,25 @@ ORDER BY five_min
 ```
 Note that the above cast to TIMESTAMP converts the time to local time according
 to the server's timezone setting.
+
+---
+
+### `last()` and `first()`
+
+The `last()` and `first()` aggregates allow you to get the value of one column as orderer by another. For example, `last(temp, time)` will return the latest temperate value based on time within an aggregate group.
+
+**Required arguments**
+
+|Name|Description|
+|---|---|
+| `value` | The value to return (anyelement) |
+| `time` | The timestamp to use for comparison (TIMESTAMP/TIMESTAMPTZ or integer type)  |
```

Comment:
Is it possible to make "order" column optional, and use the hypertable's time-partitioning column by default?